app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -519,6 +532,23 @@ private void configureRightCard() {
         rightCardMapButton.setOnClickListener(v -> presenter.openCoordinateMap());
     }
 
+    public void configurePrevButton(){
+        prevTitleDecs.setOnClickListener((View v) -> {
+            BasicKvStore storePrevTitleDesc = new BasicKvStore(this, "prevTitleDesc");
```

Comment:
use `defaultKvStore` instead of creating new storage.

Reply:
okay I'll implement all your points.